prime-router/src/main/kotlin/secrets/AzureSecretService.kt

```diff
@@ -25,7 +25,7 @@ internal object AzureSecretService : SecretService() {
 
     override fun fetchSecretFromStore(secretName: String): String? {
         val azureSafeSecretName = secretName.toLowerCase().replace("_", "-")
-        return secretClient.getSecret("functionapp-${azureSafeSecretName}")?.let {
+        return secretClient.getSecret("functionapp-$azureSafeSecretName")?.let {
             return it.value
         }
```

Comment:
KTLint kept making this change. I let it win.

Reply:
Hmm.  I think I changes this exact same line in my PR.   One of us will feel conflicted...

Reply:
Yup